index.html

```diff
@@ -1,11 +1,61 @@
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>CALYX</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cruxui-lib.netlify.app//Components/main.css"
+    />
+    <link rel="stylesheet" href="style.css">
+    <title>CALYX-ECOMM</title>
+  </head>
+  <body>
+    <div class="rootContainer">
+      <div class="navbar bg-primary p-2" style="z-index: 100">
+        <input id="menu-icon" type="checkbox" />
+        <label class="menu-label" for="menu-icon"></label>
```

Comment:
you have added label but I don't think it is displaying any thing in UI..
So please remove it if it is not useful!

Reply:
Its functionality is critical for collapsible navbar behavior. Ui is managed by css

Reply:
ok